.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
What do you think of `.prettierrc.js` instead? It would match eslint and mocha formats?

Reply:
With 2.0.1 we can go back to the old format.